modules/python3/bindings/include/inviwopy/util/pyglmhelper.h

```diff
@@ -228,11 +251,21 @@ struct ExposePortsFunctor {
             pybind11::bind_vector<std::vector<T>>(m, vectorName);
             exposeStandardDataPorts<std::vector<T>>(m, vectorName);
 
-        } else {
+        } else if constexpr (rank == 1 && util::extent_v<T> <= 4) {
+            constexpr auto N = util::extent_v<T>;
             const auto prefix = glm::detail::prefix<V>::value();
             const auto vectorName = fmt::format("{}vec{}Vector", prefix, N);
             pybind11::bind_vector<std::vector<T>>(m, vectorName, pybind11::buffer_protocol{});
             exposeStandardDataPorts<std::vector<T>>(m, vectorName);
+
+        } else if constexpr (rank == 2 && util::extent_v<T, 0> <= 4 &&
+                             util::extent_v<T, 0> == util::extent_v<T, 1>) {
```

Comment:
only symmetric matrices?

Reply:
Yes, since this is not "free" I did not want to go crazy will all permutations... unless we have a need for them...